numpy/lib/tests/test_arraysetops.py

```diff
@@ -34,7 +34,24 @@ def test_intersect1d(self):
         assert_array_equal(c, ed)
 
         assert_array_equal([], intersect1d([], []))
-
+        
+        #return_indices
```

Comment:
May as well add a `def test_intersect1d_indices(self)` here, which creates a new test rather than making the existing one longer